script/source-setup/gradle

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -e
+
+if [ -z "$(which gradle)" ]; then
```

Comment:
Oh hmmm if this isn't setting anything up I'm not sure that there's any reason to have this script 🤔 .  I think I misunderstood your previous comment about whether you should create this script.

Maybe it can install a gradlew executable to the `test/fixtures/gradle` subfolders?

I don't know, I guess that can be re-evaluated as a followup.

Reply:
I removed it, not sure download a version of gradlew is worth it.